src/libs/PolicyUtils.ts

```diff
@@ -276,6 +276,11 @@ function goBackFromInvalidPolicy() {
     Navigation.navigate(ROUTES.SETTINGS_WORKSPACES);
 }
 
+/** Get a tax with given ID from policy */
+function getTaxByID(policy: OnyxEntry<Policy>, taxID: string): TaxRate | undefined {
+    return policy?.taxRates?.taxes?.[taxID ?? ''];
```

Comment:
```suggestion
    return policy?.taxRates?.taxes?.[taxID];
```
Should be enough since taxID is always string